suite-native/module-device-onboarding/src/hooks/useOnDeviceOnboardingFinishedRedirect.tsx

```diff
@@ -0,0 +1,46 @@
+import { useCallback } from 'react';
+import { useSelector } from 'react-redux';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { selectHasBitcoinOnlyFirmware, selectIsAnyNetworkEnabled } from '@suite-common/wallet-core';
+import {
+    AppTabsRoutes,
+    AuthorizeDeviceStackRoutes,
+    DeviceOnboardingStackParamList,
+    DeviceOnboardingStackRoutes,
+    HomeStackRoutes,
+    RootStackParamList,
+    RootStackRoutes,
+    StackToStackCompositeNavigationProps,
+} from '@suite-native/navigation';
+
+type NavigationProps = StackToStackCompositeNavigationProps<
+    DeviceOnboardingStackParamList,
+    DeviceOnboardingStackRoutes,
+    RootStackParamList
+>;
+
+export const useOnDeviceOnboardingFinishedRedirect = () => {
```

Comment:
Since it uses navigation, would call it differently:
```suggestion
export const useOnDeviceOnboardingFinishedNavigation = () => {
```

Reply:
Fixed